pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -328,9 +328,9 @@ pyramid_oereb:
     # This is a multilingual value. In the minimum, the value for the default language has to be defined.
     base_data:
         text:
-          de: Daten der amtlichen Vermessung, Stand {0}.
-          fr: Données de la mensuration officielle, état actuel {0}
-          it: Dati della misurazione ufficiale, stato attuale {0}
+          de: Daten der amtlichen Vermessung. Stand der amtlichen Vermessung: {0}.
+          fr: Données de la mensuration officielle, état de la mensuration officielle: {0}
+          it: Dati della misurazione ufficiale, stato al {0}
```

Comment:
Shouldn't it be something like this:
"Dati della misurazione ufficiale. Stato della misurazione ufficiale: {0}"

But I'am really not an italien speaker :))

Reply:
The literal translation you propose is ok. "Dati della misurazione ufficiale. Stato della misurazione ufficiale: {0}." Our business person, proposed another translation, we use in our internal branch: "Dati della misurazione ufficiale, stato al {0}". Feel free to put your translation, that is also correct.

Reply:
Thank you for cooperation, @vvmruder!

Reply:
Thanks for clarification